_targets/build.project.armv7a9-zynq7000

```diff
@@ -83,10 +85,10 @@ b_build_target() {
 
 
 b_flash_image() {
-	local flash_path="${PREFIX_BOOT}/flash-armv7a9-zynq7000.bin"
+	local flash_path="${PREFIX_BOOT}/flash-armv7a9-zynq7000-${TARGET_PROJECT}.bin"
```

Comment:
In this case You can use `${TARGET}` instead of `armv7a9-zynq7000-${TARGET_PROJECT}.`